week07/seoayeong-crud-api/src/main/java/practice/seoayeongcrudapi/member/controller/MemberController.java

```diff
@@ -0,0 +1,35 @@
+package practice.seoayeongcrudapi.member.controller;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.*;
+import practice.seoayeongcrudapi.member.controller.dto.request.CreateMemberRequest;
+import practice.seoayeongcrudapi.member.controller.dto.request.UpdateMemberRequest;
+import practice.seoayeongcrudapi.member.controller.dto.response.MemberResponse;
+import practice.seoayeongcrudapi.member.service.MemberService;
+
+@RestController
+@RequiredArgsConstructor
+public class MemberController {
+
+    private final MemberService memberService;
+
+    @PostMapping("/members")
+    public void signUp(@RequestBody CreateMemberRequest createMemberRequest) {
+        memberService.signUp(createMemberRequest);
+    }
+
+    @GetMapping("/members/{id}")
```

Comment:
```suggestion
    @GetMapping("/members/{memberId}")
```
크게 중요한 것은 아니지만, 명시해줘도 좋을 것 같네요~